fs/parsetime.go

```diff
@@ -0,0 +1,106 @@
+package fs
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+)
+
+// Time is a time.Time with some more parsing options
+type Time time.Time
+
+// For overriding in unittests.
+var (
+	timeNowFunc = func() time.Time { return time.Now() }
+)
+
+// Turn Time into a string
+func (t Time) String() string {
+	if !t.IsSet() {
+		return "off"
+	}
+	return time.Time(t).Format(time.RFC3339Nano)
+}
+
+// IsSet returns if the time is not zero
+func (t Time) IsSet() bool {
+	return !time.Time(t).IsZero()
+}
+
+// parse the date as time in various date formats
+func parseTimeDates(date string) (t time.Time, err error) {
+	var instant time.Time
+	for _, timeFormat := range timeFormats {
+		instant, err = time.Parse(timeFormat, date)
```

Comment:
This should be `time.ParseInLocation(timeFormat, date, time.Local)`

See the very recently fixed 887cccb2c17fa5f9c70ce32bc4c949e5906897a7

In fact you you'll want to update your tests based on that too.

I think you should probably get `parseDurationDates` to call this function so we don't duplicate the logic! I'd move this function to `parseduration.go` to be with the list of times also.